Kconfig.zephyr

```diff
@@ -40,6 +40,8 @@ source "ext/Kconfig"
 
 source "tests/Kconfig"
 
+osource "$(PROJECT_BINARY_DIR)/Kconfig.modules"
```

Comment:
Kconfig.modules is not optional, so

`source "$(PROJECT_BINARY_DIR)/Kconfig.modules"`

Reply:
in principle we could decide not to generate the file if no modules are present.
But as we always create the file (but without content if there are no modules), then I adopted your suggestion.